src/models/Query.py

```diff
@@ -15,10 +14,33 @@ def __init__(self, table: Table, columns: list[Column], condition: Condition):
             columns: columns to extract from the query
             condition: conditional statement to filter the results in the query
         """
+        self.table = table
+        self.columns = columns
+        self.condition = condition
 
     def to_SQL_string(self) -> str:
         """
         Transforms the query into SQL code.
         Returns:
             A string with the SQL code equivalent to the query.
         """
+        select_clause = ''
```

Comment:
Definir la retribución de todas las columnas como el comportamiento por defecto.

```suggestion
        select_clause = 'SELECT *'
```